aws-storage-s3/src/main/java/com/amplifyframework/storage/s3/operation/AWSS3StorageDownloadFileOperation.kt

```diff
@@ -37,6 +37,10 @@ import java.util.concurrent.ExecutorService
 /**
  * An operation to download a file from AWS S3.
  */
+@Deprecated(
+    "Class should not be public and explicitly cast to. " +
+        "Internal usages are moving to AWSS3StorageDownloadFileOperationV2"
```

Comment:
Should this read AWSS3StoragePathDownloadFileOperation?

Reply:
Correct. These are all being renamed. I will update.